### ISSUE TITLE
torch.manual_seed(args.seed)

### ISSUE THREAD

**Issue by wutong4012** [ahangchen/torch_base#5]
在train.py文件中第21行torch.manual_seed(args.seed)，
但是在options.py文件中好像并没有设置args.seed...

**Comment by wutong4012** [comment]
除此之外，在data_entry.py问津中的第17行train_dataset = get_dataset_by_type(args)，
由于get_dataset_by_type函数中is_train默认为False，所以这里应该添加参数为True吧。
即train_dataset = get_dataset_by_type(args, True)

**Comment by ahangchen** [comment]
已修复，也欢迎直接提交pull request一起完善这个repository
